### ISSUE TITLE
Low-level objects don't expose useful bits to pythonland

### ISSUE THREAD

**Issue by bocajnotnef** [dib-lab/khmer#695]
While working in oxli and attempting to decompose the scripts so that they can be more useful within an ipython notebook/interpreter I found that a lot of low-level objects from khmerland are not visible to python in any useful context. Examble: hashbits/tables. There isn't a good way to access the hashtable itself as an object or tagsets, graphs, etc. because the hashbit/table just outputs things to files on its own without exposing itself to python.

TL;DR in @mr-c's words: Scripts and khmer itself should separate processing from IO.

**Comment by ctb** [comment]
Need more specifics; closing 'til then.
